ui/css/treeherder-pinboard.css

```diff
@@ -151,24 +140,12 @@
   text-decoration: underline;
 }
 
-/* Spin button suppression on chrome (bug 1045611) */
-#pinboard-related-bugs form input[type=number]::-webkit-inner-spin-button {
-  -webkit-appearance: none;
-  margin: 0;
-}
-
-/* Spin button suppression on firefox (bug 1045611) */
-#pinboard-related-bugs form input[type=number] {
-  -moz-appearance: textfield;
-}
-
```

Comment:
I believe this can be removed entirely, at least on OSX. Perhaps someone can check Chrome/Firefox on Linux or Windows to ensure no number spinners appear in the bug number field on those platforms.